mflowgen/Tile_MemCore/construct.py

```diff
@@ -315,6 +315,16 @@ def construct():
 
   if pwr_aware:
       pwr_aware_gls.update_params( { 'design_name': parameters['design_name'] }, True )
+     
+      init.extend_postconditions(         ["assert 'Clamping logic structure in the SBs and CBs is not maintained' in File( 'mflowgen-run.log' )"] )
```

Comment:
Can you change 'Clamping logic structure in the SBs and CBs **is not maintained**' to 'Clamping logic structure in the SBs and CBs **is maintained**'. I had committed the update after the PR, so not sure if it was grabbed.

Reply:
Ankita changed this herself in future commit